src/prefect/agent/fargate/agent.py

```diff
@@ -25,111 +32,35 @@ class FargateAgent(Agent):
         - aws_session_token (str, optional): AWS session key for connecting the boto3
             client. Defaults to the value set in the environment variable
             `AWS_SESSION_TOKEN`.
-        - task_role_arn (str, optional): AWS Amazon Resource Name (ARN) of the AWS
-            Identity and Access Management (IAM) role that grants containers in the
-            task permission to call AWS APIs on your behalf. Defaults to the value set
-             in the environment variable `TASK_ROLE_ARN`.
-        - execution_role_arn (str, optional): AWS Amazon Resource Name (ARN) of the
-            task execution role that containers in this task can assume. Defaults to
-             the value set in the environment variable `EXECUTION_ROLE_ARN`.
         - region_name (str, optional): AWS region name for connecting the boto3 client.
             Defaults to the value set in the environment variable `REGION_NAME`.
-        - cluster (str, optional): The Fargate cluster to deploy tasks. Defaults to the
-            value set in the environment variable `CLUSTER`.
-        - subnets (list, optional): A list of AWS VPC subnets to use for the tasks that
-            are deployed on Fargate. Defaults to the value set in the environment
-            variable `SUBNETS`.
-        - security_groups (list, optional): A list of security groups to associate with
-            the deployed tasks. Defaults to the value set in the environment variable
-            `SECURITY_GROUPS`.
-        - repository_credentials (str, optional): An Amazon Resource Name (ARN) of the
-            secret containing the private repository credentials. Defaults to the value
-            set in the environment variable `REPOSITORY_CREDENTIALS`.
-        - assign_public_ip (str, optional): Whether the task's elastic network interface
-            receives a public IP address. Defaults to the value set in the environment
-            variable `ASSIGN_PUBLIC_IP` or `ENABLED` otherwise.
-        - task_cpu (str, optional): The number of cpu units reserved for the container.
-            Defaults to the value set in the environment variable `TASK_CPU` or `256`
-            otherwise.
-        - task_memory (str, optional): The hard limit (in MiB) of memory to present to
-            the container. Defaults to the value set in the environment variable
-            `TASK_MEMORY` or `512` otherwise.
+        - **kwargs (dict, optional): additional keyword arguments to pass to boto3 for
```

Comment:
You're certain there are no kwarg overlaps here?

Reply:
There are but they're handled in the `_parse_kwargs` function